### PR TITLE
Refactor json serialiation helpers into a new package

### PR DESCRIPTION
**Pull request by bingenito**
Add new MorganStanley.Fdc3.Json package for serialization helpers that are useful for .NET desktop agents to interop with web.

### CODE REVIEW

File: src/Fdc3.Json/MorganStanley.Fdc3.Json.csproj [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        <VersionSuffix>alpha.1</VersionSuffix>
+		<Product>fdc3-dotnet</Product>
+		<SignAssembly>true</SignAssembly>
+		<AssemblyOriginatorKeyFile>..\keypair.snk</AssemblyOriginatorKeyFile>
+		<PackageReadmeFile>README.md</PackageReadmeFile>
+        <TargetFramework>netstandard2.0</TargetFramework>
+        <Nullable>enable</Nullable>
+        <LangVersion>8.0</LangVersion>
+        <RepositoryUrl>https://github.com/morganstanley/fdc3-dotnet</RepositoryUrl>
+        <Description>.NET Standard 2.0 declarations to implement concrete FDC3 compatible .NET desktop agents and usage of intents/contexts.</Description>
```
Comment (bingenito) [inline src/Fdc3.Json/MorganStanley.Fdc3.Json.csproj:16]: Needs unique description as it is a separate nuget

File: src/Fdc3.Json/MorganStanley.Fdc3.Json.csproj [hunk 2]
```diff
@@ -0,0 +1,45 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<RootNamespace>MorganStanley.Fdc3.Json</RootNamespace>
+		<AssemblyName>MorganStanley.Fdc3.Json</AssemblyName>
+		<VersionPrefix>2.0.0</VersionPrefix>
+        <VersionSuffix>alpha.1</VersionSuffix>
+		<Product>fdc3-dotnet</Product>
```
Comment (bingenito) [inline src/Fdc3.Json/MorganStanley.Fdc3.Json.csproj:8]: Needs unique product as it is a separate nuget

File: src/Fdc3.Json/Serialization/Fdc3CamelCaseNamingStrategy.cs [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+using Newtonsoft.Json.Serialization;
+
+namespace MorganStanley.Fdc3.Json.Serialization
+{
+    public class Fdc3CamelCaseNamingStrategy : CamelCaseNamingStrategy
+    {
+        public override string GetPropertyName(string name, bool hasSpecifiedName)
+        {
+            switch (name)
```
Comment (psmulovics) [inline src/Fdc3.Json/Serialization/Fdc3CamelCaseNamingStrategy.cs:23]: Later we can move this to pattern matching if we want

**Review by psmulovics — APPROVED** [review]

**Review by kruplm — APPROVED** [review]

File: src/Fdc3.Json/MorganStanley.Fdc3.Json.csproj (hunk 2 above)
Comment (bingenito) [reply src/Fdc3.Json/MorganStanley.Fdc3.Json.csproj:8]: Will be updated in subsequent PR

File: src/Fdc3.Json/MorganStanley.Fdc3.Json.csproj (hunk 1 above)
Comment (bingenito) [reply src/Fdc3.Json/MorganStanley.Fdc3.Json.csproj:16]: Will be updated in subsequent PR

**Pull request merged**